main.go

```diff
@@ -258,6 +264,9 @@ func getTemporaryCredentials(awsConfig aws.Config) (Credentials, error) {
 		SecretAccessKey: *sessionCreds.Credentials.SecretAccessKey,
 		Token:           *sessionCreds.Credentials.SessionToken,
 		Expiration:      string(sessionExpiration),
+		Code:            "Success",
+		LastUpdated:     time.Now().UTC().String(),
+		Type:            "AWS-HMAC",
```

Comment:
This is what the value is in the documentation. Not sure if it should be something like `imds-credential-server`